File: Python/Problem 2: Even Fibonacci Numbers.py
def fiboEvenSum(n):
    sumEven = 0
    first = 1
    last = 1
    i = 1
    
    while(i < n):
        resultFib = first + last

        if(resultFib % 2 == 0):
            sumEven += resultFib

        first = last
        last = resultFib
        
        i += 1

    return sumEven


print(fiboEvenSum(43))

File: JS/Problem 2: Even Fibonacci Numbers.js
function fiboEvenSum(n){

	const sum = (a, b) => a + b;
	
	let sumEven = 0;
	let first = 1;
	let last = 1;

	for(let i = 1; i < n; i++){
		
		let resultFib = sum(first, last);
		
		if(resultFib % 2 === 0) sumEven += resultFib;

		first = last;
		last = resultFib;
	}
	return sumEven;
}


console.log(fiboEvenSum(43));
